packages/esm-patient-conditions-app/src/conditions/conditions-form.test.tsx

```diff
@@ -111,25 +111,29 @@ describe('ConditionsForm: ', () => {
       conditionSearchInput = screen.getByRole('searchbox', { name: /Enter condition/i });
     });
 
-    it('renders related condition concepts when the user types in the searchbox', () => {
+    it('renders a list of related condition concepts when the user types in the searchbox', async () => {
       mockSearchConditionConcepts.mockReturnValueOnce(of(searchedCondition).pipe(delay(1000)));
 
       expect(screen.queryByRole('menuitem', { name: /Headache/i })).not.toBeInTheDocument();
       expect(screen.queryByDisplayValue('Headache')).not.toBeInTheDocument();
 
       fireEvent.change(conditionSearchInput, { target: { value: 'Headache' } });
 
+      await waitForLoadingToFinish();
+
       expect(screen.getByDisplayValue('Headache')).toBeInTheDocument();
     });
 
-    it('renders an error message when no matching conditions are found', () => {
-      mockSearchConditionConcepts.mockReturnValueOnce(of([]));
+    it('renders an error message when no matching conditions are found', async () => {
+      mockSearchConditionConcepts.mockReturnValueOnce(of([]).pipe(delay(1000)));
```

Comment:
Does this add a full second to test execution time? Why?

Reply:
I need to delay the emission of values from the observable to allow the loading state enough time to render in the test environment. 1000ms is overkill as you say, so I've lowered it down to 10ms.